api/src/test/java/org/openmrs/util/OpenmrsUtilUnitTest.java

```diff
@@ -137,4 +147,330 @@ public void join_shouldReturnListElementsJoinedBySeparatorIfGivenSeparatorIsNull
 
 		assertThat(OpenmrsUtil.join(moduleList, null), is("module1nullmodule2"));
 	}
+
+	@Test
+	public void isStringInArray_shouldReturnTrueIfStringIsInArray() {
+
+		String[] array = new String[]{"element1","element2","element3"};
+
+		assertTrue(OpenmrsUtil.isStringInArray("element1", array));
+	}
+
+	@Test
+	public void isStringInArray_shouldReturnFalseIfStringIsNotInArray() {
+
+		String[] array = new String[]{"element1","element2","element3"};
+
+		assertFalse(OpenmrsUtil.isStringInArray("element4", array));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldReturnFalseIfHiNormalIsNull() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(null);
+
+		assertFalse(OpenmrsUtil.isInNormalNumericRange(5.67f, concept));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldReturnFalseIfLowNormalIsNull() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setLowNormal(null);
+
+		assertFalse(OpenmrsUtil.isInNormalNumericRange(5.67f, concept));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldReturnTrueIfValueIsInRange() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(10.34d);
+		concept.setLowNormal(3.67d);
+
+		assertTrue(OpenmrsUtil.isInNormalNumericRange(5.67f, concept));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldReturnFalseIfValueIsAboveRange() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(4.97d);
+		concept.setLowNormal(3.67d);
+
+		assertFalse(OpenmrsUtil.isInNormalNumericRange(5.67f, concept));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldReturnFalseIfValueBelowAboveRange() {
```

Comment:
rename to
`isInNormalNumericRange_shouldReturnFalseIfValueBelowRange`

since it cannot be BelowAbove at the same time. maybe just a typo :)